src/main/java/net/sf/jabref/model/database/EntrySorter.java

```diff
@@ -24,9 +24,16 @@
 
 public class EntrySorter {
 
-    // guarded by itself
     private final List<BibEntry> entries;
 
+
+    /**
```

Comment:
I would remove this comment completely, as this comment can be automatically derived from the JavaCode.